Framework/Geometry/src/Instrument/ComponentInfoBankHelpers.cpp

```diff
@@ -66,8 +66,14 @@ bool isSaveableBank(const ComponentInfo &compInfo, const size_t idx) {
     if (parentType != Beamline::ComponentType::Rectangular &&
         parentType != Beamline::ComponentType::Structured &&
         parentType != Beamline::ComponentType::Grid) {
-      // if component at index is not a tube then identify it as a bank
+      // check if component is sub-assembly of tube, if so returns false
```

Comment:
"a sub-assembly" of a tube? an eight-pack is a super assembly comprising tubes.

Reply:
that correct, i meant to say that